core/coin.go

```diff
@@ -0,0 +1,44 @@
+package core
+
+import (
+	"errors"
+	"strings"
+)
+
+// DefaultCurrencyDivisibility - decimals for price
+const DefaultCurrencyDivisibility uint32 = 1e8
+
+var (
+	ErrUnsupportedCurrencyCode = errors.New("unsupported currency code")
+	SupportedCurrencies        = []Currency{
+		Currency("btc"),
+		Currency("bch"),
+		Currency("eth"),
+		Currency("ltc"),
+		Currency("zec"),
+		Currency("tbtc"),
+		Currency("tbch"),
+		Currency("teth"),
+		Currency("tltc"),
+		Currency("tzec"),
+	}
+)
+
+// Currency describes
```

Comment:
This godoc comment should be completed, and all the other exported identifiers should probably have them as well (when we have golint in CI it'll be required for all changes).

Reply:
I think I started down this path and got distracted elsewhere. Will do.